Makefile

```diff
@@ -9,13 +9,10 @@ clean:
 	# lcs.cpp and blocks.cpp are generated by cython
 	rm -rf build dist dragnet/*.pyc dragnet/*.so dragnet/lcs.cpp dragnet/blocks.cpp dragnet.egg-info dragnet/features/*.pyc dragnet/features/_kohlschuetter.cpp dragnet/features/_readability.cpp dragnet/features/_weninger.cpp
 
-test: nose
+test: pytest
 
-nose:
-	nosetests --exe --cover-package=dragnet --with-coverage --cover-branches -v --cover-erase
-
-unittest:
-	python -m unittest discover -s test
+pytest:
+	pytest --cov=dragnet --cov-report=term --cov-branch -v test
```

Comment:
For the sake of discussion: would it be better to set this up in a `setup.cfg`? I think something roughly like this would get us the same thing:
```
[tool:pytest]
testpaths = test
addopts = --cov=dragnet

[coverage:run]
branch = True
```
And then we could just run tests with `pytest` instead of `make test`.

That said, it doesn't gain us too much except for being a more standard approach.

Reply:
I was aiming for a minimal port — no changes besides those required for `unittest` + `nose` => `pytest` — so this was out of scope. That said, I'm open to making this change if it's advisable. (I don't have any prior experience with `setup.cfg`.)

Reply:
Nope, that makes total sense to me (and was kinda what I figured). Just thought I'd mention it in case it seemed useful :)